### ISSUE TITLE
Value assertions when importing from different modules is breaking transformer

### ISSUE THREAD

**Issue by piotrwitek** [ikatyang/dts-jest#299]
Hey @ikatyang,
I really love concept of this library and I'm trying to port my typescript library [`typesafe-actions`](https://github.com/piotrwitek/typesafe-actions) to `dts-jest`, but unfortunately I have found out an issue with value testing.

Everything works great when testing only types (without transpilation involved) but I would like to test values as well in my tests and it's not working when I'm importing a function (that I try to test) from a different module.

<img width="811" alt="screenshot 2018-12-25 at 10 59 21 pm" src="https://user-images.githubusercontent.com/739075/50426963-56c1b800-089b-11e9-9235-bdf61406aae2.png">

I have added an example test case so you can try to reproduce it yourself.
You can find it here in my fork. You just need to run `npm run test-integration`:
https://github.com/piotrwitek/dts-jest/tree/modules_not_working_demo

I think it have something to do with the transpilation process which don't translate import statements correctly.

I would appreciate any help with that issue!

**Comment by ikatyang** [comment]
See https://github.com/ikatyang/dts-jest/pull/300#issuecomment-449906761.

**Comment by piotrwitek** [comment]
@ikatyang Thank you for help.
I still think the modules example I did would be much more useful than your contrived example. The problem is when testing for real you're mostly importing modules and not testing global objects, and when people will start to use it for testing values they will most probably have the same problem I did.

The best I would suggest is to update jest config example in README to include the change you did to fix the issue for me:
```
"transform": {
    "example[^]*\\.ts$": "<rootDir>/transform",
    "\\.ts$": "ts-jest" // add this to transpile modules when using value assertions
  },
```

Anyway great work with this lib!
